webapp/channels/src/utils/keyboard.ts

```diff
@@ -50,3 +48,12 @@ export function clickOnMenuButton(buttonId: string) {
 
     menuButton.click();
 }
+
+export function clickOnMenuOverlay() {
```

Comment:
Nothing to change, but I wonder if we might want this to be called something like `dismissMenus` or be located in `actions/views`. I'm trying to think where I'd want this code to look if menus were part of Redux like modals are

Reply:
moved it to menu component and renamed it to dismissMenu